libvast/src/format/csv.cpp

```diff
@@ -167,8 +168,15 @@ reader::reader(const caf::settings& options, std::unique_ptr<std::istream> in)
   if (in != nullptr)
     reset(std::move(in));
   using defaults = vast::defaults::import::csv;
-  opt_.separator
+  opt_.separator = defaults::separator[0];
+  auto seperator_option
     = get_or(options, "vast.import.csv.separator", defaults::separator);
+  if (seperator_option.size() != 1)
+    VAST_WARN("{} encountered invalid vast.import.csv.separator '{}'; must be "
+              "a single character",
```

Comment:
Is this really a restriction?

Reply:
Yes, unless we want to rewrite large parts of the parser. We exclude the separator character from the allowed character list in many places, which isn't trivial to exchange with an arbitrary-length string.

Reply:
Okay, so nothing in principle that would hold us back, but too much scope-creep for this PR.